pandas/tests/reshape/test_concat.py

```diff
@@ -1010,6 +1010,21 @@ def test_append_missing_column_proper_upcast(self, sort):
         assert appended['A'].dtype == 'f8'
         assert appended['B'].dtype == 'O'
 
+    def test_append_empty_frame_to_series_with_dateutil_tz(self):
+        # GH 23682
+        date = Timestamp('2018-10-24 07:30:00', tz=dateutil.tz.tzutc())
+        s = Series({'date': date, 'a': 1.0, 'b': 2.0})
+        df = DataFrame(columns=['c', 'd'])
+        result = df.append(s, ignore_index=True)
+        expected = DataFrame([[np.nan, np.nan, 1., 2., date]],
+                             columns=['c', 'd', 'a', 'b', 'date'])
+        # These columns get cast to object after append
+        object_cols = ['c', 'd', 'date']
+        expected.loc[:, object_cols] = expected.loc[:, object_cols].astype(
+            object
+        )
+        assert_frame_equal(result, expected)
```

Comment:
@mroeschke what were the expected dtypes of result and expected here?

On master, everything is object, which may not have been intended.

On #24024, expected has `datetime64[ns, tzutc()]` dtype for the `date` column.

Reply:
@TomAugspurger `datetime64[ns, tzutc()]` would be the correct dtype. I may have cast to object due to an append bug that existed prior. Feel free to change this test in #24024